tests/testthat/test-lines.R

```diff
@@ -16,3 +16,36 @@ test_that("lines don't have fill", {
   })
   expect_equal(xml_attr(xml_find_one(x, ".//polyline"), "fill"), "none")
 })
+
+
+test_that("lines lty becomes stroke-dasharray", {
+  x <- xmlSVG({ plot(1:3,lty=1,type="l") })
```

Comment:
Could you remove some of the duplication by making a function that retrieves the dasharray with a function